CSL JSON.js

```diff
@@ -56,7 +56,10 @@ function detectImport() {
 }
 
 function doImport() {
-	if(!parsedData) parseInput();
+	//We do call parseInput always because otherwise multiple async translator
+	//calls might interfer together.However, it may be that parsedData is
+	//already set from detectImport.
+	parseInput(); 
```

Comment:
(though I don't think it's related to the underlying issue with the DOI translator)